pom.xml

```diff
@@ -24,6 +24,24 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.32</version>
         </dependency>
+        <dependency>
```

Comment:
Please, remove one of the doubled dependency

Reply:
Ok done!